float32 ifm_time
int64 modulation_frequency
int64 illu_mode
int64 reset_duration
bool median_filter
bool mean_filter
int64 trigger_mode
int64 sampling_mode
int64 integration_time_single
int64 integration_time_double
int64 delay_time
float32 unambiguous_range
float32 evaluation_time
int64 data_port
string ip
int64 xml_port

